### DISCUSSION TITLE
provide monochrome version of the Floorp logo

### DISCUSSION THREAD

**Discussion by dieub56** [Floorp-Projects/Floorp#811] [Feature Request]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

I'm using a Gnome extension to display and control the current media player, there is an option to disable icon color but Floorp doesn't seem to support this. It just displays the logo image regardless, the screenshots show the issue.

Spotify (color disabled) :
![Screenshot from 2024-01-29 12-13-43](https://github.com/Floorp-Projects/Floorp/assets/47575489/18420fa2-20af-4ccf-a248-4213181ed0a2)

Spotify (color enabled) :
![Screenshot from 2024-01-29 12-17-04](https://github.com/Floorp-Projects/Floorp/assets/47575489/c3dbb4b6-af1f-44ca-9381-4e5a6076adb3)

Floorp behavior (unaffected by this option) :
![Screenshot from 2024-01-29 12-13-35](https://github.com/Floorp-Projects/Floorp/assets/47575489/bd7ab68d-292a-4770-bb27-4ba4b1b2330f)

### To Reproduce

This happens with the [media controls extension](https://extensions.gnome.org/extension/4470/media-controls/) on Gnome Shell 45.3 (Fedora).

### Expected behavior

 Spotify allows switching from a color to monochrome (vector ?) version of their logo, but Floorp does not. See the screenshots above.

### OS Info

Fedora Linux 39 (Workstation Edition)

### Floorp Version

11.8.2

**Comment by Ryosuke-Asano** [comment]
This is not a bug. I'll forward it to feature request.

**Comment by CutterKnife** [comment]
The monochrome logo is the one implemented by the extension.

**Reply by dieub56** [reply]
I thought apps maybe had differents of the same logo and the extension switched between those when / if they were available. You're saying the spotify monochrome version of the logo was made as part of the extension directly ?

**Reply by dieub56** [reply]
There are several issues open in the repository for this extension about this, as far as I'm aware my understanding is correct regarding apps making the icons available to the player. Here is an issue discussing this in more detail how the extension retrieves the icon files : https://github.com/sakithb/media-controls/issues/47

Clearly some apps handles icons properly and others dont, in my case Chromium had the same issue as Floorp but Firefox work properly, like Spotify. I'm sure other extensions make use of these icons in Gnome, so this would hopefully benefit other Floorp users who enjoy a nice themed desktop.

**Reply by CutterKnife** [reply]
> I thought apps maybe had differents of the same logo and the extension switched between those when / if they were available.

Are you referring to GNOME's "Symbolic App Icons"?

**Reply by dieub56** [reply]
I'm not all that familiar with Gnome but that sounds like what I tried to describe, and what the various issues on that extension are refering to.
